docs/reference/index-modules/translog.asciidoc

```diff
@@ -71,7 +71,9 @@ update, or bulk request. This setting accepts the following parameters:
   The translog stores all operations that are not yet safely persisted in Lucene
   (i.e., are not part of a Lucene commit point). Although these operations are
   available for reads, they will need to be replayed if the shard was stopped
-  and had to be recovered. This setting controls the maximum total size of these
-  operations, to prevent recoveries from taking too long. Once the maximum size
-  has been reached a flush will happen, generating a new Lucene commit point.
-  Defaults to `512mb`.
+  and had to be recovered.
+  This setting controls the maximum total size of these operations to prevent
+  recoveries from taking too long. Once the maximum size has been reached, a flush
+  will happen, generating a new Lucene commit point. *Defaults to* `10 GB`.
+  The translog size will never exceed `1%` and will never be smaller than `10 MB`,
+  regardless of the specified size.
```

Comment:
Not sure we need to surface this info in the docs. It might be too low-level of a detail. But even if we want to display this info, we should do it in the Translog main section above. I'd defer to @danielharada's opinion about technical accuracy.

Reply:
The 1% of disk space limitation applies to any nodes that have less than 1000 GB of disk allocated.  In Cloud, on our storage optimized hardware profile (the default choice), 5 out of 7 node size options would fall into this category.

The 10 MB lower limit would only apply to nodes with less than 1 GB of disk space.  I don't see any node size options in Cloud that could trigger this limit.

So, I feel like the 1% of disk usage limit would be applicable often enough that I would expect to see it in the docs, but the 10 MB lower limit should be a pretty extreme edge case that is fine to leave out.

I would clarify:
>The translog size will never exceed `1%` *of the disk size*

because with the current wording I don't know what the 1% value is applying to.  Feel free to re-word that as appropriate.

Reply:
Thank you for your feedback!
I’ve removed the part about the 10 MB limit and relocated the remaining details to the general description of the Translog, where I believe they fit more appropriately.